hpOneView/resources/hypervisors/hypervisor_cluster_profiles.py

```diff
@@ -102,3 +103,26 @@ def get_compliance_preview(self):
         """
         compliance_uri = "{0}/compliance-preview".format(self.data["uri"])
         return self._helper.do_get(compliance_uri)
+
+    def delete(self, timeout=-1, softDelete=False, force=False):
```

Comment:
variables name style should be consistent.

Reply:
changed softDelete to soft_delete